packages/wallet-sdk/src/stores/sub-accounts/utils.ts

```diff
@@ -0,0 +1,27 @@
+import { SubAccountInfo } from './store.js';
+
+export function assertSubAccountInfo(info: unknown): asserts info is SubAccountInfo {
+  if (typeof info !== 'object' || info === null) {
+    throw new Error('info is not an object');
```

Comment:
created a backlog [ticket](https://linear.app/coinbase/issue/BW-1072/[sdk]-sub-account-flow-to-throw-standard-errors) to throw standard error in SDK. we can perhaps throw `rpc.internal` here

Reply:
Good callout. I'll update in a future PR